Makefile

```diff
@@ -0,0 +1,12 @@
+build:
+	go build -o osbuild-composer ./cmd/osbuild-composer/
+	go build -o osbuild-worker ./cmd/osbuild-worker/
+
+install:
```

Comment:
This should depend on `build`, shouldn't it?

Reply:
Yes, in theory. But in that case `build` should probably depend on the source files so we don't rebuild it every time even without changing it. But I don't know it sth like `$(shell find . -name '*.go')` is appropriate for go packages.

Reply:
Fair point. Leave it as is.